index.html

```diff
@@ -403,6 +429,45 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
       </dl>
     </section>
   </section>
+  <section id="rtcresolutionrestriction">
+    <h3>
+      The {{RTCResolutionRestriction}} dictionary.
+    </h3>
+    <pre class="idl">dictionary RTCResolutionRestriction {
+    unsigned long maxWidth;
+    unsigned long maxHeight;
+};</pre>
+    <section id="rtcresolutionrestriction-attributes">
+      <h2>Dictionary {{RTCResolutionRestriction}} Members</h2>
+      <dl data-link-for="RTCResolutionRestriction" data-dfn-for="RTCResolutionRestriction" class="dictionary-members">
+        <dt>
+          <dfn data-idl>maxWidth</dfn> of type <span class="idlMemberType">unsigned long</span>
+        </dt>
+        <dd>
+          <p>The maximum width that frames will be encoded with without getting
+          downscaled. The size comparison is orientation agnostic. When scaling
```

Comment:
Define "orientation agnostic" - either that X is always X no matter what the CVO rotation parameter says, or that if width > height, width is X and height is Y, or vice versa.

Reply:
Defined further down, it's the swap meaning that's intended. I worry about this in a world of tons of vertical video.

Reply:
In a world of [VideoFrame](https://w3c.github.io/webcodecs/#videoframe) and effects processing, I think this just needs to work when we inspect the frame WxH. Or is there a different way to do this that is better?

Reply:
The intent is that you shouldn't need to call setParamters() every time the orientation changes in order to get the same scaling factors

Reply:
If I express this in terms of [primary orientation](https://w3c.github.io/mediacapture-main/#dfn-primary-orientation) instead, then specifying 1280x720 should work regardless of orientation, but it would in addition also allow me to enter a 720x1280 and force it to fit into that? Is that correct + what we should do?

Reply:
Feedback from meeting: the UA should calculate pixel count from W and H and use that as the restriction

Reply:
Also: Jan-Ivar liked the name scaleResolutionDownTo better, will update

Reply:
Calculating pixel count internally as an implementation for "orientation agnostic" does not work when the restriction is not the same aspect ratio as the frame.

Example: frame is 1280x720 and I want to restrict it to 360 in height, but I care less about width and say {maxWidth:2000, maxHeight:360}. What should happen to get 360p and maintain aspect ratio is a 640x360 frame, but if we internally calculate a really large pixel count (2000*360) I will end up with a frame that is much taller than 360.

I think we need to either swap W and H on the fly (current PR), or not be orientation agnostic, or swap W and H but do so based on orientation metadata rather than trying to infer it from input frame's dimensions.

Reply:
As discussed, I removed the talk about orientation since the rotation is separate from what is width and height meaning the frame is 1280x720 regardless of orientation and we can always say width and height without any tweaks to this.

I also changed the name back to scaleResolutionDownTo and updated the text about when not to send duplicate encodings.